src/StripeTests/Infrastructure/StringUtilsTest.cs

```diff
@@ -48,5 +48,29 @@ public void SecureEquals()
                     StringUtils.SecureEquals(testCase.data.a, testCase.data.b));
             }
         }
+
+        [Fact]
+        public void ContainsWhitespace()
+        {
+            var testCases = new[]
+            {
+                new { data = "sk_test_123", want = false },
+                new { data = "sk_test_4eC39HqLyjWDarjtT1zdp7dc", want = false },
+                new { data = "abc", want = false },
+                new { data = "sk-test-123", want = false },
+                new { data = string.Empty, want = false },
```

Comment:
why are we okay with this one?

Reply:
This tests the utility method called `ContainsWhitespace`, and empty strings don't contain whitespace ;)

For API keys specifically, if the string is empty, it will be handled like `null`: https://github.com/stripe/stripe-dotnet/pull/1520/files#diff-04df6fa6018c18be44475e1f882f6509R106